keyboards/v60_type_r/keymaps/iso/keymap.c

```diff
@@ -0,0 +1,54 @@
+/* This is the default ISO layout provided by the KBP V60 Type R
+* as depicted on the stock keycaps.
+*/
+#include QMK_KEYBOARD_H
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+  /* Keymap 0: Default Layer (Qwerty)
+   * ,-----------------------------------------------------------.
+   * |Esc|  1|  2|  3|  4|  5|  6|  7|  8|  9|  0|  -|  =|   Bs  |
+   * |-----------------------------------------------------------|
+   * |Tab  |  Q|  W|  E|  R|  T|  Y|  U|  I|  O|  P|  [|  ]| Entr|
+   * |-------------------------------------------------------|   |
+   * |Caps   |  A|  S|  D|  F|  G|  H|  J|  K|  L|  ;|  '|  #|   |
+   * |-----------------------------------------------------------|
+   * |LShif|  ||  Z|  X|  C|  V|  B|  N|  M|  ,|  .|  /| RShift  |
+   * |-----------------------------------------------------------|
+   * |Ctrl|Gui |Alt |      Space              |Fn0 |Gui |App|Ctrl|
+   * `-----------------------------------------------------------'
+   */
+  [0] = LAYOUT_60_iso( \
+      KC_ESC, KC_1, KC_2, KC_3, KC_4, KC_5, KC_6, KC_7, KC_8, KC_9, KC_0, KC_MINS, KC_EQL, KC_BSPC, \
+      KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_LBRC, KC_RBRC, KC_ENT, \
```

Comment:
2\. Moving `KC_ENT` to home row:

```suggestion
      KC_TAB, KC_Q, KC_W, KC_E, KC_R, KC_T, KC_Y, KC_U, KC_I, KC_O, KC_P, KC_LBRC, KC_RBRC, \
```